This is required for the module to load directly, e.g. manifypods

--- origsrc/Gtk2-WebKit-0.03/lib/Gtk2/WebKit.pm	2008-08-28 00:35:23.000000000 -0500
+++ src/Gtk2-WebKit-0.03/lib/Gtk2/WebKit.pm	2008-12-14 23:23:21.978250000 -0600
@@ -4,6 +4,8 @@
 use warnings;
 use base qw/DynaLoader/;
 
+use Gtk2;
+
 our $VERSION = '0.03';
 
 sub dl_load_flags { 0x01 }
